### PR TITLE
Remove licence related methods

### PR DESCRIPTION
**Pull request by davidbasalla**
This app does not need to talk to Licencify anymore, since [we now do it from Frontend instead](https://github.com/alphagov/frontend/pull/1141). That means we can remove licence related methods as well as references to the LocalService model (Licences were the only ones still using it). We also remove a failing test relating to LocalTransactions and LocalServices as we a.) don't request local services for local transactions and b.) have migrated LocalTransactions to be read from the content store instead of the content api.

Note we could remove any code relating to migrated formats (i.e. help pages, answers, simple smart answers, guides, local transactions) but we'll do that once all Publisher formats have been migrated.

### CODE REVIEW

File: lib/presenters/artefact_presenter.rb [hunk 1]
```diff
@@ -141,26 +138,6 @@ def nodes
...
 
-  def licence
-    return {} unless @artefact.licence
-
-    {
-      "licence" => ArtefactLicencePresenter.new(@artefact.licence).present
-    }
-  end
-
-  def local_service
```
Comment (whoojemaflip) [inline lib/presenters/artefact_presenter.rb]: How is this related to Licences?

Comment (davidbasalla) [inline lib/presenters/artefact_presenter.rb]: Licences and LocalTransactions used to inject the local service information into the response hash. Both have been changed to not use the local service info anymore (LocalTransaction has been migrated and Licences now gets the relevant information straight from Licensify) so it can be removed from the presenter. I added some text to the commit message, does that make it clear? Or let me know if you prefer another commit for this...

**Comment by whoojemaflip** [comment]
:taco:
